apps/src/templates/sectionsRefresh/sections-refresh.module.scss

```diff
@@ -9,7 +11,7 @@
 
 .buttonRow {
   // ensure we have height when we only have our toggle (which is floated)
-  min-height: 50;
+  min-height: 50px;
```

Comment:
Fixing styling from the pr before when I didn't understand how css works

Reply:
I have made this mistake so many times 🙃